SCENE-SCRAPABLE.md

```diff
@@ -98,6 +98,9 @@ czechstreets.com|Czechav.yml
 czechsupermodels.com|Czechav.yml
 czechtaxi.com|Czechav.yml
 czechwifeswap.com|Czechav.yml
+czechvr.com|CzechVR.yml
+czechvrfetish.com|CzechVRFetish.yml
+czechvrcasting.com|CzechVRCasting.yml
```

Comment:
move `czechvrcasting.com|CzechVRCasting.yml` up one position so that the order is correct

Reply:
Thanks for the feedback.  Bone-headed mistake.  It's fixed with this latest push.